addon/index.js

```diff
@@ -47,16 +48,19 @@ const SERVER_ENVIRONMENTS = {
   local: {
     BASE_URL: 'http://testpilot.dev:8000',
     TESTPILOT_PREFIX: 'testpilot.addon.LOCAL.',
+    WHITELIST_URLS: 'https://www.mozilla.org/*,about:home',
```

Comment:
Cool! Seems silly to duplicate them right now. But, hopefully once we have some integration testing that involves the add-on, we can patch this to include local URL origins relevant to those tests.